### PR TITLE
link to the indexing audit reference

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

File: lighthouse-core/audits/seo/is-crawlable.js [hunk 1]
```diff
@@ -63,8 +63,9 @@ class IsCrawlable extends Audit {
       name: 'is-crawlable',
       description: 'Page isn’t blocked from indexing',
       failureDescription: 'Page is blocked from indexing',
-      helpText: 'The "Robots" directives tell crawlers how your content should be indexed. ' +
-      '[Learn more](https://developers.google.com/search/reference/robots_meta_tag).',
+      helpText: 'Search engines are less likely to include your pages in search results ' +
```
Comment (rviscomi) [inline lighthouse-core/audits/seo/is-crawlable.js]: Same comment as https://github.com/google/WebFundamentals/pull/5857#discussion_r171977228 regarding "less likely"

**Comment by rviscomi** [comment]
@paulirish should I be merging as soon as I approve?

**Comment by rviscomi** [comment]
Progress on #4355

**Comment by paulirish** [comment]
@rviscomi for changes this trivial, sure. :) 

we do squash merges btw. and typically delete the body of the comment.

**Pull request merged**